modules/virt-creating-a-primary-cluster-udn.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-primary-udn.adoc
+
+:_mod-docs-content-type: PROCEDURE                                
+[id="virt-creating-a-primary-cluster-udn_{context}"]                                  
+= Creating a primary cluster-scoped user-defined network
+
+You can connect multiple namespaces to the same primary user-defined network (UDN) to achieve native tenant isolation.
+
+.Prerequisites
+. You have access to the cluster as a user with `cluster-admin` privileges.
+. You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+. Create a `ClusterUserDefinedNetwork` object to specify the custom network configuration:
++
+.Example `ClusterUserDefinedNetwork` manifest
+[source,yaml]
+----
+kind: ClusterUserDefinedNetwork
+metadata:
+  name: cudn-l2-net # <1>
+spec:
+  namespaceSelector: # <2>
+    matchExpressions:
+    - key: kubernetes.io/metadata.name
+      operator: In
+      values:
+        - "mynamespace"
+        - "theirnamespace"
+  template:
+    topology: Layer2 # <3>
+    role: Primary # <4>
+    mtu: 9000
+    subnets: ["10.0.0.0/24"]
+    excludeSubnets: ["10.0.0.100/26"]
```

Comment:
this is wrong. 

It recently changed. Please check [this](https://github.com/maiqueb/fosdem2025-p-udn/blob/afe9e49a264f7f6ca98c86854742c45c24bf57cd/manifests/cluster-wide-network/01-udn.yaml#L42) example of a correct c-udn.

@ormergi do we expose the `excludeSubnets` attribute ?... I don't see it in the [API reference](https://ovn-kubernetes.io/api-reference/userdefinednetwork-api-spec/#clusteruserdefinednetwork) ... which we know is **not** up-to-date :crying_cat_face:

Reply:
We dont for l2 & l3 topologies, for the incoming localnet topology it will have excludeSubnets

Reply:
@maiqueb we should specify persistent IPs for VMs right?

Reply:
Oh for sure we want `Persistent` ipam lifecycle. 

Good catch !

Reply:
I have updated the YAML based on [this doc](https://github.com/maiqueb/fosdem2025-p-udn/blob/afe9e49a264f7f6ca98c86854742c45c24bf57cd/manifests/cluster-wide-network/01-udn.yaml#L42)